scripts/attach_contract_versions.sh

```
@@ -0,0 +1,14 @@
+OUT_DIR="artifacts"
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)